colab_evaluation.py

```diff
@@ -0,0 +1,226 @@
+# coding=utf-8
+# Copyright 2020 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Version of evaluation utilities intended for use in lower memory environments, such as colab.
+
+This version of the evaluation code leverages the fact that the vast majority of label-example predictions 

```

Comment:
nit: this line uses label-example and the next line uses example-label. I'd use example-label.